### ISSUE TITLE
Error while generating dynamic children for Data Table

### ISSUE THREAD

**Issue by Ravi92sr** [FlutterFlow/flutterflow-issues#1063]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I'm generating dynamic children for a data table from a variable.
I get compilation error while trying to run the project

### Expected Behavior

Project should run without any errors.

### Steps to Reproduce

1. Create a new page
2. Add a Data Table
3. Generate the children for the table from a variable
4. Run the project

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITFfzPHlwItPscte7q6ILMdFvCQWJlscTL4viutuZy0gGJDtBIh/P+DsUxZvOdTnYHlhHGCWrEQz0dbtv9jxOO05Fw6ZQYRB/qpqewLMY3q8RcmzPbiKW2gmI8ZhFUys3ImNmyVRPtpucnBkw2aMeeevXjXsGZO/Zwh9f6PHaOI=

### Context

Not able to run the project because of this issue. There is another data table in the project which works fine.

### Visual documentation

![image](https://github.com/FlutterFlow/flutterflow-issues/assets/123063001/69acdbc2-7c57-430f-88b9-9a672f750f72)

### Environment

```markdown
- FlutterFlow version: v3.1
- Platform:Web
- Browser name and version: Chrome Version 114.0.5735.90 (Official Build) (64-bit)
- Operating system and version affected: Ubuntu
```

**Comment by hariprasadms** [comment]
Hi @Ravi92sr - Thanks for issue posting. I did try the above steps in my test project. It is working fine. I am able to generate datable and its row data by mapping a collection and its data to it. Not able to re produce the issue.

I looked at your project as well. It is challenge to pinpoint the issue because there are more pages and multiple data-tables with TabView widgets. Set up a blank project with the exact configuration of widgets, and reproduce the issue. Send blank project bug report code. I can revisit the issue once you have added these details. Thanks

https://github.com/FlutterFlow/flutterflow-issues/assets/17435678/aeaa787c-cf28-49c8-b5f0-80969aed7242
